trollmoves/tests/test_server.py

```diff
@@ -123,3 +124,12 @@ def test_process_notify(Message):
         with file_cache_lock:
             assert "/topic/20200428_1000_foo.tif" in file_cache
             assert len(file_cache) == 1
+
+
+class TestDeleter(unittest.TestCase):
```

Comment:
Should there be a test that the file is actually deleted?

Reply:
There should indeed, but I didn't change anything about this in the PR...